packages/next/src/client/components/error-boundary.tsx

```diff
@@ -100,19 +100,19 @@ export class ErrorBoundaryHandler extends React.Component<
 }
 
 export default function GlobalError({ error }: { error: any }) {
+  const digest: string | undefined = error?.digest
   return (
     <html>
       <head></head>
       <body>
         <div style={styles.error}>
           <div>
             <h2 style={styles.text}>
-              Application error: a client-side exception has occurred (see the
-              browser console for more information).
+              {`Application error: a ${
+                digest ? 'server' : 'client'
+              } side exception has occurred (see your server logs for more information).`}
```

Comment:
```suggestion
              }-side exception has occurred (see your server logs for more information).`}
```

Should we keep the hyphen?